Tests/Marketplace/search_and_install_packs.py

```diff
@@ -168,8 +168,79 @@ def search_pack(client: demisto_client, prints_manager: ParallelPrintsManager, p
         lock.release()
 
 
+def install_nightly_packs(client, host, prints_manager, thread_index, packs_to_install, request_timeout=999999):
+    """
+    Install content packs on nightly build.
+    Args:
+        client(demisto_client): The configured client to use.
+        host (str): The server URL.
+        prints_manager (ParallelPrintsManager): Print manager object.
+        thread_index (int): the thread index.
+        packs_to_install (list): A list of the packs to install.
+        request_timeout (int): Timeout settings for the installation request.
+
+    Returns:
+
+    """
+    packs_to_install_str = ', '.join([pack['id'] for pack in packs_to_install])
```

Comment:
```suggestion
    packs_to_install_str = '\n '.join([pack['id'] for pack in packs_to_install])
```

Reply:
Its better to leave the ', '. This way the print is not too long